Resources/Private/Language/de.locallang_db.xlf

```diff
@@ -31,6 +31,9 @@
 				<source>Usergroup Access Rights</source>
 				<target>Zugriffsrechte für Benutzergruppen</target>
 			</trans-unit>
+			<trans-unit id="tx_tea_domain_model_product_tea.hidden">
+				<source>Sichtbar</source>
```

Comment:
This needs to be the `target` (with the `source` copied from the `locallang_db.xml`).

Reply:
Yea, my fault, fixing right now.